plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env bash
 
+if [ ! -z ${CI+y} ]; then
```

Comment:
@adimoldovan: FYI, if you remember in 2024, I tried to speed up the script, but we had to revert. Here is another attempt, aiming at our CI and staying compatible with other environments.

In case we still have an environment which can be affected by the change, please let me know.